### DISCUSSION TITLE
Feature: Vertical Lifebar

### DISCUSSION THREAD

**Discussion by cyliamargatroid** [ikemen-engine/Ikemen-GO#886] [Feature Request] [answered]
MUGEN usually uses horizontal lifebars, what if there's a Vertical type of lifebar?

**Comment by ghost** [comment]
That should be pretty easy to add, perhaps in a small 0.98 update?
Vertical Lifebars sound cool, there is one in Star Fox 64 as an example of what it would look like.
Dunno why MUGEN never had that, perhaps they weren't aware of such styles?

**Comment by rakieldev** [comment, marked as answer]
Added by #2437
